File: JavaScript Solutions/1036.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let A = parseFloat(lines.shift()),
    B = parseFloat(lines.shift()),
    C = parseFloat(lines.shift()),
    D = Math.pow(B, 2) - 4 * A * C,
    rD = Math.sqrt(D);
    
if (A===0 || D<0)
    console.log("Impossivel calcular");
else {
    const R1 = ((-B + rD)/(2*A)).toFixed(5);
    const R2 = ((-B - rD)/(2*A)).toFixed(5);
    
    console.log("R1 = "+R1+"\nR2 = "+R2);
}

File: Python Solutions/1036.py
dados = input().split()
A= float(dados[0])
B= float(dados[1])
C= float(dados[2])

D= B**2 - 4*A*C

if (D<0 or A==0):
    print("Impossivel calcular")
    
else:
    R1= (B*(-1) + D**0.5)/(2*A)
    R2= (B*(-1) - D**0.5)/(2*A)
    print(f"R1 = {R1:.5f}")
    print(f"R2 = {R2:.5f}")       
